Positions/SeniorSoftwareEngineer.md

```diff
@@ -1,36 +1,102 @@
 
-N.b. Not every single one of these criteria needs to be satisfied to be considered senior. But seniority comprises, in some combination or other, the majority of these.
+_Not every single one of these criteria needs to be satisfied to be considered a full JSE. But the role comprises, in some combination or other, the majority of these._
 
-### Mastery in Drupal, our primary platform
-- May have a specialty: systems integration, presentation layer
-- Mastery within that specialty, and a comfortable Drupal generalist
-- Knowledge of best practices: coding standards, core architecture, module vs. OO architecture, theming layer
-- Able to argue for when to do things in Drupal, and when not to
+Grading scale for each section uses the following:
 
+- Needs Opportunity
+- Needs Significant Work
+- Basic
+- Good
+- Great
+- Expert
 
-### Technical leadership
-- Regular consultation with other developers on architecture, best practices, documentation
-- Presentation of ideas, technologies, demos for tech team and full staff
-- Influence on how we work and what we work on
-- Mentor relationship with other engineers
+### Technical Skill/Knowledge
+- Understands the broad architecture of the entire system
+- Knowledge of best practices: coding standards, architecture, testing, validation, data recovery
+- Mastered several technologies which allows for evaluation/selection from new and known approaches to successfully implement projects of moderate complexity
+- Designs and implements subsystems of the software architecture that can scale and be reliable with minimal technical oversight 
+- Understands/Can setup, manage and monitor application and infrastructure/build (i.e., deployment pipeline, logging, application monitoring and alerts, rollbacks, code quality checks)
+- Handles product releases
+- Makes significant contributions to our test coverage
+- Effective at debugging
+- Demonstrated ability to integrate with external APIs
+- Comfortable with database tuning & administration
+- Understands and is able to setup automated functional, load/performance, and integration tests
+- Familiarity with and contributes to caching strategy
+- Able to effectively QB at least one area of the codebase
+- Familar/Comfortable with more than one area of the codebase
 
-### Automated testing
-- Facility with unit and end-to-end testing
-- Significant contributions to our test coverage
-- Could be more front-end (Wraith, QUnit) or back-end (Drupal unit/web tests, Behat)
+### Impact
+- Scopes and stages work into well-defined milestones to avoid a monolithic deliverable
+- Regularly delivers software on-time and is constantly working to make accurate estimates and deliver on those estimates
+- Contributes to a framework or development practice. Has enough experience here to have a point of view, and can argue for systemic improvements
+- Possesses empathy with the user of the software they are producing and uses that empathy to guide decision-making  
```

Comment:
"decision-making" -> "decision making"

Reply:
Updated both occurrences in the spreadsheet as well